model/src/pyrenew/process/periodiceffect.py

```diff
@@ -61,6 +61,11 @@ def __init__(
             broadcast_type="tile",
         )
 
+        self.set_timeseries(
+            timeseries_start=0,
+            timeseries_unit=1,
+        )
```

Comment:
Although the broadcaster class in the library talks about period length/unit, I opted to say that samples using `PeriodicBroadcaster` return data at the daily level starting observation time 0. What do you think about this?

Reply:
Any reason not to make this configurable? Or are you just thinking about defaults? My instinct would be to make it mandatory to specify at minimum the unit, given that it's a broadcasting operation.

Reply:
I like that, probably make both arguments mandatory then: `timeseries_start` and `timeseries_unit`.